zerver/models/realms.py

```diff
@@ -187,6 +187,7 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     MAX_REALM_SUBDOMAIN_LENGTH = 40
     MAX_REALM_REDIRECT_URL_LENGTH = 128
 
+    DEFAULT_REALM_DESCRIPTION = "The coolest place in the universe."
```

Comment:
If I understand it correctly, this wasn't marked for translation earlier either, and neither is it now.

Reply:
I have now marked it using `gettext_lazy`.

Reply:
I have marked it for translation now.